src/main/java/org/openstreetmap/josm/plugins/pt_assistant/utils/HelpUtils.java

```diff
@@ -0,0 +1,81 @@
+package org.openstreetmap.josm.plugins.pt_assistant.utils;
+
+import static org.openstreetmap.josm.tools.I18n.tr;
+
+import org.openstreetmap.josm.gui.MainApplication;
+import org.openstreetmap.josm.gui.MapFrame;
+import org.openstreetmap.josm.gui.widgets.JMultilineLabel;
+import org.openstreetmap.josm.tools.GBC;
+import org.openstreetmap.josm.tools.ImageProvider;
+import org.openstreetmap.josm.data.preferences.BooleanProperty;
+
+import javax.swing.*;
+import javax.swing.border.CompoundBorder;
+import javax.swing.border.EmptyBorder;
+import javax.swing.border.EtchedBorder;
+import java.awt.Color;
+import java.awt.Cursor;
+import java.awt.Font;
+import java.awt.GridBagLayout;
+
+/**
+ * Help messages that are displayed on the bottom of the map
+ *
+ * @author sudhanshu2
+ */
+public class HelpUtils extends JPanel {
+
+    private static final HelpUtils ClassInstance = new HelpUtils();
+    private static final BooleanProperty doNotShow = new BooleanProperty("message.ptassistant.helputil", false);
+
+    private HelpUtils() {
+        /* Empty to prevent initialization */
+    }
+
+    public void addToJOSM(String message) {
+        if (doNotShow.get() == Boolean.FALSE) {
```

Comment:
!=

Mind, that booleans are not always instance-equal. So it is better to use:
`!Boolean.FALSE.equals(doNotShow.get())`

And settings do not have a `null` value, so you could simplify:
`!doNotShow.get()`

Reply:
I also don't really find it helpful to not use non-negated phrases to name boolean variables for better readability :stuck_out_tongue_winking_eye::

`!doNotShow.get()`
vs
`!hideHelpUtil.get()` or `showHelpUtil.get()`

From the property name `message.ptassistant.helputil` it's not really intuitive that `false` means "show the messages" and `true` means "don't show the messages".